<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-06RECOG" failures="1" tests="3" name="Test" time="19.571" errors="0" timestamp="2023-09-23T17:17:27 IST">
  <testcase classname="TestCases.VerifyLogin" name="Invalidlogin" time="1.041"/>
  <testcase classname="TestCases.VerifyLogin" name="login" time="0.653">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	error message is not valid expected [Please check your username. If you still can&amp;apos;t log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
error message is not valid expected [Please check your username. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at Resources.CommonMethods.handleAssertions(CommonMethods.java:17)
at TestCases.VerifyLogin.login(VerifyLogin.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="TestCases.VerifySignUp" name="signup" time="6.556"/>
</testsuite> <!-- Test -->
